scripts/README.md

```diff
@@ -0,0 +1,544 @@
+
+	classDef storage fill:#A3C1DA,stroke:#A3C1DA,stroke-width:2px,color:#000000;
+	class C,E,G,J storage;
+
+	classDef collection fill:#D3A6A1,stroke:#D3A6A1,stroke-width:2px,color:#000000;
+	class A,B collection;
+
+	classDef compute fill:#A9C0C9,stroke:#A9C0C9,stroke-width:2px,color:#000000;
+	class D,F,K,L,N compute;
+
+	classDef visualization fill:#E8D5A6,stroke:#E8D5A6,stroke-width:2px,color:#000000;
+	class M,O,P visualization;
+```
+
+## File flow details
+**Naming conventions**
+-   Experiment folder name:
+	- `< Proposal Prefix>-< 5 digit proposal number>_< first part of email address>/`
+-   h5 files in experiment folder:
+	- `< YYYYMMDD>_< HHMMSS>_< user defined string>.h5`
+-   h5 files for tiled scans:
+	- `< YYYYMMDD>_< HHMMSS>_< user defined string>_x<##>y<##>.h5`
+
+**data832 Raw Location**
+- `data/raw/< Experiment folder name convention>/< h5 files>`
+
+**data832 Scratch Location**
+- `data/scratch/globus_share/< Experiment folder name convention>/rec< dataset>/< tiffs>`
+- `data/scratch/globus_share/< Experiment folder name convention>/rec< dataset>.zarr/`
+
+**ALCF Raw Location**
+- `/eagle/IRIBeta/als/bl832/raw/< Experiment folder name convention>/< h5 files>`
+
+**ALCF Scratch Location**
+- `/eagle/IRIBeta/als/bl832/scratch/< Experiment folder name convention>/rec< dataset>/< tiffs>`
+- `/eagle/IRIBeta/als/bl832/scratch/< Experiment folder name convention>/rec< dataset>.zarr/`
+
+## Globus Compute Flow
+
+The preferred method for scheduling jobs on ALCF is through Globus Compute. This `reconstruction_wrapper()` function contains the code registered with Globus, which runs the reconstruction on ALCF. The commands call two Python programs located in the IRIBeta project folder on Polaris. Here is an example of code that is registered and run in the Globus Compute Flow:
+```python
+def  reconstruction_wrapper(rundir,  h5_file_name,  folder_path):
+	...
+	# Run reconstruction.py
+	# Output: A folder of tiff images
+    command =  f"python /eagle/IRIBeta/als/example/globus_reconstruction.py {h5_file_name}  {folder_path}"
+    recon_res = subprocess.run(command.split("  "),  stdout=subprocess.PIPE,  stderr=subprocess.PIPE)
+	...
+	# Run tiff_to_zarr.py
+	# Output: .zarr directory
+    file_name = h5_file_name[:-3]  if h5_file_name.endswith('.h5')  else h5_file_name
+    command =  f"python /eagle/IRIBeta/als/example/tiff_to_zarr.py /eagle/IRIBeta/als/bl832_test/scratch/{folder_path}/rec{file_name}/"  # --zarr_directory /path/to/storage/"
+    zarr_res = subprocess.run(command.split("  "),  stdout=subprocess.PIPE,  stderr=subprocess.PIPE)
+```
+### Reconstruction
+
+The `globus_reconstruction.py` script in this flow is a slightly modified version of Dula Parkinson's tomography reconstruction code.  The modifications update the function's input parameters and set proper file/folder permissions for the reconstructed data.
+
+Location on Polaris: `/eagle/IRIBeta/als/example/globus_reconstruction.py`
```

Comment:
These files are just placed on polaris? If they're not under version control, perhaps we can put them in the repo under `/script/polaris`?  Same with the tiff_to_zarr.py below